apps/emqx_exhook/src/emqx_exhook.app.src

```diff
@@ -1,6 +1,6 @@
 {application, emqx_exhook,
  [{description, "EMQ X Extension for Hook"},
-  {vsn, "git"},
+  {vsn, "4.3.0"},
```

Comment:
Why not git?

Reply:
git is for the whole project.
apps may not always change in every release.
starting from 4.3.0, we will use semver for app changes. (apps in the umbrella project)